deploy/gppkg/madlib.spec.in

```diff
@@ -7,7 +7,7 @@
 BuildRoot:      @MADLIB_GPPKG_RPM_SOURCE_DIR@
 Summary:        Apache MADlib for @GPDB_VARIANT@ Database
 License:        @CPACK_RPM_PACKAGE_LICENSE@
-Name:           Apache_MADlib
+Name:           madlib
```

Comment:
This change doesn't appear to version or upgrade related. What is this change for (GPDB specific gppkg spec file)? Aren't we still delivering and rpm named Apache MADlib?